libs/wingsdk/test/simulator/__snapshots__/simulator.test.ts.snap

```diff
@@ -91,17 +91,7 @@ exports[`run single test > happy path 1`] = `
 
 exports[`run single test > test failure 1`] = `
 {
-  "error": "Error: test failed
-    at Handler.handle ([abs])
-    at Object.exports.handler ([abs])
-    at [abs]
-    at Script.runInContext (node:vm:<sanitized>)
-    at Object.runInContext (node:vm:<sanitized>)
-    at [src]/shared/sandbox.ts:<sanitized>
-    at new Promise (<anonymous>)
-    at Sandbox.call ([src]/shared/sandbox.ts:<sanitized>)
-    at Object.withTrace ([src]/simulator/simulator.ts:<sanitized>)
-    at TestRunner.runTest ([src]/target-sim/test-runner.inflight.ts:<sanitized>)",
+  "error": undefined,
```

Comment:
Oops

Reply:
My bad! I've made this mistake before so you'd think I'd remember that error is just a string...

Reply:
I'll just use the first line